docs/advanced/input_files/input-main.md

```diff
@@ -276,6 +276,7 @@
     - [hubbard\_u](#hubbard_u)
     - [yukawa\_potential](#yukawa_potential)
     - [yukawa\_lambda](#yukawa_lambda)
+    - [uramping](#uramping)
```

Comment:
I suggest you to add description of uramping in converge.md or construct_H.md docs.